e2e/steps_file.js

```diff
@@ -146,6 +149,7 @@ module.exports = function () {
         ... conditionalSteps(!litigantInPerson, [
           () => respondentRepresentedPage.enterRespondentRepresented('respondent1', 'yes'),
           () => defendantSolicitorOrganisation.enterOrganisationDetails('respondent1'),
+          () => defendantSolicitorServiceAddress.enterOrganisationServiceAddress(),
```

Comment:
`SecondDefendantSolicitorServiceAddress` page needs to be covered on e2e tests as well

Reply:
done